server/resources/ProspectsResources.py

```diff
@@ -1,18 +1,24 @@
 from flask_restful import Resource, reqparse
 from models.ProspectModel import ProspectModel
+import json
 
 from flask_jwt_extended import (jwt_required, get_jwt_identity)
 
+from app import db
+
 upload_parser = reqparse.RequestParser()
-upload_parser.add_argument('prospects', type=str, location='json', required=True)
+upload_parser.add_argument('prospects', type=list, location='json', required=True)
```

Comment:
I created a reqparser generator in utils, might want to use that.
it would just take ["prospects"] as the argument, give it a look!

Reply:
This needs to have type=list and location='json' to parse correctly